src/graphics/Screen.cpp

```diff
@@ -379,10 +379,15 @@ static void drawTextMessageFrame(OLEDDisplay *display, OLEDDisplayUiState *state
         display->fillRect(0 + x, 0 + y, x + display->getWidth(), y + FONT_HEIGHT_SMALL);
         display->setColor(BLACK);
     }
-    display->drawStringf(0 + x, 0 + y, tempBuf, "From: %s", (node && node->has_user) ? node->user.short_name : "???");
-    if (config.display.heading_bold) {
-        display->drawStringf(1 + x, 0 + y, tempBuf, "From: %s", (node && node->has_user) ? node->user.short_name : "???");
+
+    if(config.display.heading_bold) {
+        display->drawStringf(1 + x, 0 + y, tempBuf, "           From: %s", (node && node->has_user) ? node->user.short_name : "???");
```

Comment:
Please use the inserstion on both print statements, heading_bold works by printing the same text twice with a 1 pixed offset. The way you set it up the timestamp is not in bold print.

Reply:
Done in commit 65c0b8e.